Dockerfile.ci

```diff
@@ -18,6 +18,8 @@ ENV SPARSE_VERSION=$SPARSE_VERSION
 ARG PROTOC_VERSION=21.7
 ENV PROTOC_VERSION=$PROTOC_VERSION
 ARG WGET_ARGS="-q --show-progress --progress=bar:force:noscroll"
+ARG FVP_BASE_REVC_VERSION=11.24_11
+ARG FVP_BASE_AEMV8R_VERSION=11.24_11
```

Comment:
Please add the corresponding `ENV`s for these (see https://github.com/zephyrproject-rtos/docker-image/pull/164).